cbindings/pedersen.cc

```diff
@@ -113,7 +115,11 @@ static void process_compute_pedersen_commitments(struct sxt_bls12_381_g1_compres
   auto num_generators = populate_exponent_sequence(sequences, descriptors);
 
   auto backend = cbn::get_backend();
-  basct::cspan<cg1t::element_p2> generators_span{generators, num_generators};
+
+  // Convert from affine to projective elements
+  memmg::managed_array<cg1t::element_p2> generators_span(num_generators);
+  cg1t::batch_to_element_p2(basct::span<cg1t::element_p2>{generators_span.data(), num_generators},
+                            basct::cspan<cg1t::element_affine>{generators, num_generators});
```

Comment:
Same comments as https://github.com/spaceandtimelabs/blitzar/pull/37/files#r1373478290

Reply:
I'm not sure how to get rid of the last awkward parameter here since there is no viable conversion from `const cg1t::element_affine *` to `basct::cspan<cg1t::element_affine>`.

```
cg1t::batch_to_element_p2(generators_p, basct::cspan<cg1t::element_affine>{generators, num_generators});
```
Do you have a suggestion to make the last parameter less awkward?

Reply:
The last parameter is fine.